Documentation/Tech Stack.md

```diff
@@ -0,0 +1,26 @@
+# Tech Stack
+- The following frameworks are popular and widespread, so future developers will be able to pick up where we left off as requested by the client.
+
+- Website needs to be usable on desktop and mobile, many users may only have one or the other. 
+
+--- **Website Hosting** ---
+Client has requested that the server is located in Canada
+ Options:
+    - Self-Hosted via a rented Canadian VPS such as [ovh](https://www.ovhcloud.com/en-ca/vps/)
+    - Cloud based dynamic hosting services such as [Amazon EC2](https://aws.amazon.com/ec2/)
+    - Cloud Based Website Hosting service such as [Amazon Lightsail](https://aws.amazon.com/lightsail/)
+
+--- **Website Tech Stack** ---
+Potential Front/Back Ends:
+1. Express.js backend 
+A backend framework that is highly used, used by companies such as PayPal, Uber, and IBM. Flexible, as features come in as plugins.
+
+2. React.js front end
+React is a library that is mainly used for creating user interfaces.
+
+3. Angular.js front end
+Angluar is a library for developing and testing single page applications. 
+
+4. Vue.js front end
+Vue is a library for building user interfaces and single page applications, made to be a lightweight version of Angular.js.
+
```

Comment:
We're trying to explain _why_ we're thinking of using these stacks, not just what they do. Also, these descriptions are much too vague. I would recommend doing some more research on each option and updating the descriptions.

It might also be wise to include the database option discussed. Please see the Discord discussion we had on this topic: https://discordapp.com/channels/758049427876217013/758049756055339058/762893766343655434